mage/query-modules/cpp/meta.md

```diff
@@ -0,0 +1,199 @@
+  - `label` : number_of_occurrences
+- `relationshipTypes: Map[string: int]` ➡ map with the following (key, value) pairs:
+  - `(:label)-[:relationship_type]->()` : number_of_occurrences
+  - `()-[:relationship_type]->(:label)` : number_of_occurrences
+  - `()-[:relationship_type]->()` : number_of_occurrences
+- `relationshipTypesCount: Map[string: int]` ➡ map with the following (key, value) pairs:
+  - `relationship_type` : number_of_occurrences
+- `stats` ➡ map which contains all of the above
+
+#### Usage:
+
+Running stats on the following graph:
+```cypher
+MERGE (a:Node {id: 0}) MERGE (b:Node {id: 1}) CREATE (a)-[:Relation1]->(b);
+MERGE (a:Node {id: 1}) MERGE (b:Node {id: 2}) CREATE (a)-[:Relation1]->(b);
+MERGE (a:Node {id: 2}) MERGE (b:Node {id: 0}) CREATE (a)-[:Relation1]->(b);
+MERGE (a:Node {id: 3}) MERGE (b:Node {id: 3}) CREATE (a)-[:Relation2]->(b);
+MERGE (a:Node {id: 3}) MERGE (b:Node {id: 4}) CREATE (a)-[:Relation2]->(b);
+MERGE (a:Node {id: 3}) MERGE (b:Node {id: 5}) CREATE (a)-[:Relation2]->(b);
+```
+
+```cypher
+CALL meta.stats_online() YIELD stats;
+```
+
+```plaintext
++-------------------------------------------------------+
+|     stats                                             |
++-------------------------------------------------------+
+|                                                       |
+|{                                                      |
+|   "labelCount": 1,                                    |
+|   "labels": {                                         |
+|      "Node": 6                                        |
+|   },                                                  |
+|   "nodeCount": 6,                                     |
+|   "propertyKeyCount": 1,                              |
+|   "relationshipCount": 6,                             |
+|   "relationshipTypeCount": 2,                         |
+|   "relationshipTypes": {                              |
+|      "()-[:Relation1]->()": 3,                        |
+|      "()-[:Relation1]->(:Node)": 3,                   |
+|      "()-[:Relation2]->()": 3,                        |
+|      "()-[:Relation2]->(:Node)": 3,                   |
+|      "(:Node)-[:Relation1]->()": 3,                   |
+|      "(:Node)-[:Relation2]->()": 3                    |
+|   },                                                  |
+|   "relationshipTypesCount": {                         |
+|      "Relation1": 3,                                  |
+|      "Relation2": 3                                   |
+|   }                                                   |
+|}                                                      |
+|                                                       |
++-------------------------------------------------------+
+```
+
+### `stats_offline()`
+
+Retrieves the graph metadata by traversing the whole graph. *stats_online* should be preferred because of the better complexity unless you don't want to use triggers.
```

Comment:
```suggestion
Retrieves the graph metadata by traversing the whole graph. `stats_online` should be preferred because of the better complexity unless you don't want to use triggers.
```